activejob/lib/active_job/test_helper.rb

```diff
@@ -117,7 +117,7 @@ def queue_adapter_for_test
     #       HelloJob.perform_later('elfassy')
     #     end
     #   end
-    def assert_enqueued_jobs(number, only: nil, except: nil, queue: nil)
+    def assert_enqueued_jobs(number, only: nil, except: nil, queue: nil, at: at)
```

Comment:
This causes a warning:

```ruby
activejob/lib/active_job/test_helper.rb:120: warning: circular argument reference - at
```

Why is this unused argument needed?

Reply:
Thanks! Addressed in https://github.com/rails/rails/pull/36881